src/test/java/seedu/exercise/logic/commands/events/EventTestUtil.java

```diff
@@ -63,6 +83,35 @@
             .withExercise(getTypicalExercises().get(3))
             .withExercise(getTypicalExercises().get(4))
             .build();
+    public static final Regime LEVEL_TWO_REGIME = new RegimeBuilder()
+            .withName(LEVEL_TWO_REGIME_NAME)
+            .withExerciseList(getTypicalExercises().subList(3, 6))
+            .build();
+    // Has the last 2 exercises of Regime 1 and first exercise of regime 2.
+    public static final Regime LEVEL_THREE_REGIME = new RegimeBuilder()
+            .withName(LEVEL_THREE_REGIME_NAME)
+            .withExerciseList(getTypicalExercises().subList(1, 4))
+            .build();
+
+    // Schedules
+    public static final Schedule LEVEL_ONE_REGIME_DATE_1_SCHEDULE = new ScheduleBuilder()
+            .withRegime(LEVEL_ONE_REGIME)
+            .withDate(DATE_1)
+            .build();
+    public static final Schedule LEVEL_TWO_REGIME_DATE_1_SCHEDULE = new ScheduleBuilder()
+            .withRegime(LEVEL_TWO_REGIME)
+            .withDate(DATE_1)
+            .build();
+    public static final Schedule LEVEL_THREE_REGIME_DATE_1_SCHEDULE = new ScheduleBuilder()
+            .withRegime(LEVEL_THREE_REGIME)
+            .withDate(DATE_1)
+            .build();
+
+    // Conflict
+    public static final Conflict LEVEL_ONE_AND_TWO_REGIME_DATE_1_CONFLICT = new ConflictBuilder()
+            .withScheduled(LEVEL_ONE_REGIME_DATE_1_SCHEDULE)
+            .withConflict(LEVEL_TWO_REGIME_DATE_1_SCHEDULE)
+            .build();
```

Comment:
may want to place test data in to `CommonTestData.java`

Reply:
I felt that it would be more organized if I place them all together in the EventsUtil. Every exercise, regime, schedule, conflict, payload, command and events here are related. It's easier to test the execution of the undo and redo that way

Reply:
Then please make them private to denote it belongs to this test class. or package private if it is used in the event package only

Reply:
Alright